src/Plugins/LevelDBStore/IO/Data/LevelDB/Native.cs

```diff
@@ -10,71 +10,79 @@
 // modifications are permitted.
 
 using System;
-using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-namespace Neo.IO.Data.LevelDB
+namespace Neo.IO.Storage.LevelDB
 {
-    public enum CompressionType : byte
+    public enum CompressionType : int
     {
-        kNoCompression = 0x0,
-        kSnappyCompression = 0x1
+        NoCompression = 0,
+        SnappyCompression = 1,
     }
 
     public static class Native
     {
         #region Logger
         [DllImport("libleveldb", CharSet = CharSet.Ansi, CallingConvention = CallingConvention.Cdecl)]
-        public static extern IntPtr leveldb_logger_create(IntPtr /* Action<string> */ logger);
+        public static extern nint leveldb_logger_create(nint /* Action<string> */ logger);
```

Comment:
Is this change needed?

Reply:
Yes, `nint` is a `dotnet` `keyword` and gets compiled in the `ilasm` that is much faster OpCode then using the `dotnet` `objects classes`